tokenomics.md

```diff
@@ -27,23 +27,21 @@ Token allocation is benchmarked to similar successful projects. Typically, such
     2. Aggregators.
     3. Users.
 
-OBX tokens will be created to power the network, and a large number of those tokens would remain unassigned in the early days of the network going live as it is critical to have a large, non-circulating token reserve that can be deployed as needed to drive growth and adoption.
-
-One point to note on the listing above is the allocation to initial funders. They include the initial developer of Obscuro.  They will receive an allocation of the Obscuro token as payment in kind for the work done to-date on the project. This is a one-time only allocation, after which funders will not be involved in either development or day to day operations of the Obscuro Foundation. The number of tokens allocated to funders is to be agreed by the Obscuro Foundation.
+One point to note on the listing above is the allocation to initial funders. They include the initial developer of Obscuro.  They receive an allocation of the Obscuro token as payment in kind for the work done to date on the project. This is a one-time only allocation, after which funders will not be involved in either development or day to day operations of the Obscuro Foundation. The number of tokens allocated to funders is to be agreed upon by the Obscuro Foundation.
```

Comment:
funders sounds very much like founders. It confuses me.
Shouldn't we find a different word, like "investors"?

Reply:
Agreed, investors.

Reply:
backers?